x-pack/test/observability_functional/apps/observability/alerts/pagination.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+const ROWS_COUNT_TO_HIDE_PAGE_SELECTOR = 10;
```

Comment:
This is a tricky variable to name! 
One suggestion I could think of is "ROWS_NEEDED_FOR_PAGINATION" but then maybe the number needs to be 11 instead of 10.

Reply:
Oh yeah that was a tricky one! I like your suggestion more

Reply:
Number should stay 10. When 10 alerts are found, then pagination appears in the screen. When it is 9, it gets hidden